src/tvheadend/ChannelTuningPredictor.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+/*
+ *      Copyright (C) 2005-2015 Team Kodi
+ *      http://www.xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, write to
+ *  the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ *  http://www.gnu.org/copyleft/gpl.html
+ *
+ */
+
+#include "entity/Channel.h"
+#include <set>
+#include <utility>
+
+namespace tvheadend
+{
+
+  namespace predictivetune
+  {
+
+    /**
+     * Used to indicate that predictive tuning failed to determine which
+     * channel to be tuned next
+     */
+    const unsigned int CHANNEL_ID_NONE = -1;
```

Comment:
Ofc, just wanted to be sure that this was intended.

One last thing: why unsigned int and not uint32_t like used elsewhere with channel ids? https://github.com/kodi-pvr/pvr.hts/pull/117/files#diff-ce6ea06b04dac3a0c1f703fe9ed2cc37R65